course-content/30_bcdata.qmd

```diff
@@ -0,0 +1,233 @@
+---
+title: "Introduction to bcdata"
+teaching: 45
+exercises: 10
+questions:
+- "Fill this in!"
+objectives:
+- "Fill this in too!"
+keypoints:
+- "Same verse!"
+---
+
+<center>
+    <img src="images/bcdata/data_catalog.png" 
+    width="80%" style="margin:auto"/>
+    <p style="text-align: center">
+        B.C. Data Catalogue
+    </p>
+</center>
+
+The British Columbia government hosts over 2000 tabular and geospatial data sets in the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/). Most data is available through the B.C. Data Catalogue under an open licence. The `bcdata` package provides a programming interface to the **geospatial** data sets of the  B.C. Data Catalogue. This allows Python users to import data directly from the B.C. Data Catalogue into their Python session. Through this functionality the bcdata package connects British Columbia government public data holdings in the B.C. Data Catalogue with the vast capabilities of Python.
+
+## Installing
+
+Unlike the rest of the packages we have downloaded (using `conda` commands), the `bcdata` package is not available for download via conda (this is often the case for smaller packages that are not widely distributed globally). This is one of those instances where we need to mix our use of `pip` and `conda`. However, the installation process proceeds in the same fashion as before. These commands can be done from an anaconda prompt, or from the command prompt (`cmd`) utility inside VS Code. 
+
+```{.bash filename='Anaconda Prompt'}
+> conda activate ds-env
+> pip install bcdata
+```
+
+As before, make sure that you are installing this package in the `ds-env` environment! To make sure that this has successfully downloaded, we can try importing it into our Python session. 
+
+```{python}
+import bcdata
+bcdata.__version__
+```
+
+## Basic Usage
+
+### Introduction to the Catalogue
+
+To use the `bcdata` package, we will need to familiarize ourselves with the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/). If we enter a search term in the catalog, it will bring up a variety of data sets that might meet our needs. However, the python `bcdata` package is only for geospatial datsets. These will typically be data types with a *resource storage format* (one of the filter options) that matches:
```

Comment:
```suggestion
To use the `bcdata` package, we will need to familiarize ourselves with the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/). If we enter a search term in the catalogue, it will bring up a variety of data sets that might meet our needs. However, the python `bcdata` package is only for geospatial datsets. These will typically be data types with a *resource storage format* (one of the filter options) that matches:
```